### ISSUE TITLE
It can't lemmatize `going` and `doing`

### ISSUE THREAD

**Issue by nukuutos** [winkjs/wink-lemmatizer#32]
Current behavior:
```js
lemmatize.verb('going') // -> going
lemmatize.verb('doing') // -> doing
```
Expected:
```js
lemmatize.verb('going') // -> go
lemmatize.verb('doing') // -> do
```
  
And I have additional question: does `winkNLP` use `wink-lemmatizer` for lemmatization under the hood? I just looking for a good solution for POS tagging and lemmatization. And I'm not good at nlp :)

**Comment by sanjayaksaxena** [comment]
Hi @nukuutos 

`WinkNLP` has zero dependencies — it does not use `wink-lemmatizer`. It will meet both of your requirements — recommend that you explore it.

Had provided a temp fix for your earlier issue, whereas it needed a lexicon level fix via `wink-lexicon`. Will roll out a proper fix.

Best,
Sanjaya

**Comment by sanjayaksaxena** [comment]
Have released v3.0.4 with the fix.
